File: documentation/user/en/query/filtering/examples/locale/locale_missing.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					hierarchyWithin(
						"categories",
						attributeEquals("code", "vouchers-for-shareholders")
					),
					entityLocaleEquals(Locale.forLanguageTag("cs"))
				),
				require(
					entityFetch(
						attributeContent("code", "name")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/locale/locale_missing.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("code", "vouchers-for-shareholders")
        		),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("cs"))
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "name")
        		)
        	)
        )
	)
);